README.md

```diff
@@ -30,6 +30,28 @@ defaultConfig {
 </manifest>
 ```
 
+- To avoid your app to crash in release mode, add a file `android/app/proguard-rules.pro` with these lines :
```

Comment:
The crash is one of our use case.

You can juste say, To use the sdk in release mode (...) otherwise, the androd a Java exception will be throw.